lib/elements/fill_stitch.py

```diff
@@ -461,10 +461,41 @@ def running_stitch_length(self):
            unit='mm',
            type='float',
            default=0.1,
-           sort_index=32)
+           sort_index=43)
     def running_stitch_tolerance(self):
         return max(self.get_float_param("running_stitch_tolerance_mm", 0.2), 0.01)
 
+    @property
+    @param('enable_random_stitches',
+           _('Randomize stitches'),
+           tooltip=_('Randomize stitch length and phase instead of dividing evenly or staggering. '
+                     'This is recommended for closely-spaced curved fills to avoid Moiré artefacts.'),
+           type='boolean',
+           select_items=[('fill_method', 'auto_fill'),
+                         ('fill_method', 'contour_fill'),
+                         ('fill_method', 'guided_fill'),
+                         ('fill_method', 'circular_fill')],
+           default=False,
+           sort_index=44)
+    def enable_random_stitches(self):
+        return self.get_boolean_param('enable_random_stitches', False)
+
+    @property
+    @param('random_stitch_length_delta_mm',
+           _('Random stitch length delta'),
+           tooltip=_('Randomize stitch length and phase instead of dividing evenly or staggering. '
```

Comment:
Can we tweak this tooltip to explain what the delta does?  Is it just the maximum amount that the running stitch length can change by, plus or minus?

Reply:
fixed. This was a bad copypaste